Source/Core/Core/NetPlayServer.cpp

```diff
@@ -648,7 +648,7 @@ bool NetPlayServer::StartGame()
 	// no change, just update with clients
 	AdjustPadBufferSize(m_target_buffer_size);
 
-	g_netplay_initial_gctime = CEXIIPL::GetGCTime();
+	g_netplay_initial_gctime = (u32)Common::Timer::GetLocalTimeSinceJan1970();
```

Comment:
Perhaps it's not a good idea to cast this to a u32.

Might cause issues after 2106.